pkg/theia/commands/policy_recommendation_run.go

```diff
@@ -0,0 +1,473 @@
+	)
+	policyRecommendationRunCmd.Flags().StringP(
+		"start-time",
+		"s",
+		"",
+		`The start time of the flow records considered for the policy recommendation.
+Format is YYYY-MM-DD hh:mm:ss in UTC timezone. No limit of the start time of flow records by default.`,
+	)
+	policyRecommendationRunCmd.Flags().StringP(
+		"end-time",
+		"e",
+		"",
+		`The end time of the flow records considered for the policy recommendation.
+Format is YYYY-MM-DD hh:mm:ss in UTC timezone. No limit of the end time of flow records by default.`,
+	)
+	policyRecommendationRunCmd.Flags().StringP(
+		"ns-allow-list",
+		"n",
+		"",
+		`List of default traffic allow namespaces.
+If no namespaces provided, Traffic inside Antrea CNI related namespaces: ['kube-system', 'flow-aggregator',
+'flow-visibility'] will be allowed by default.`,
+	)
+	policyRecommendationRunCmd.Flags().Bool(
+		"rm-labels",
+		true,
+		`Enable this option will remove automatically generated Pod labels including 'pod-template-hash',
+'controller-revision-hash', 'pod-template-generation'.`,
+	)
+	policyRecommendationRunCmd.Flags().Bool(
+		"to-services",
+		true,
+		`Use the toServices feature in ANP and recommendation toServices rules for Pod-to-Service flows,
+only works when option is 1 or 2.`,
+	)
+	policyRecommendationRunCmd.Flags().Int32(
+		"executor-instances",
+		1,
+		"Specify the number of executors for the spark application. Example values include 1, 2, 8, etc.",
+	)
+	policyRecommendationRunCmd.Flags().String(
+		"driver-core-request",
+		"200m",
+		`Specify the cpu request for the driver Pod. Values conform to the Kubernetes convention.
+Example values include 0.1, 500m, 1.5, 5, etc.`,
+	)
+	policyRecommendationRunCmd.Flags().String(
+		"driver-memory",
+		"512M",
+		`Specify the memory request for the driver Pod. Values conform to the Kubernetes convention.
+Example values include 512M, 1G, 8G, etc.`,
+	)
+	policyRecommendationRunCmd.Flags().String(
+		"executor-core-request",
+		"200m",
+		`Specify the cpu request for the executor Pod. Values conform to the Kubernetes convention.
+Example values include 0.1, 500m, 1.5, 5, etc.`,
+	)
+	policyRecommendationRunCmd.Flags().String(
+		"executor-memory",
+		"512M",
+		`Specify the memory request for the executor Pod. Values conform to the Kubernetes convention.
+Example values include 512M, 1G, 8G, etc.`,
+	)
+	policyRecommendationRunCmd.Flags().Bool(
+		"wait",
+		false,
+		"Enable this option will hold and wait the whole policy recommendation job finished.",
+	)
+	policyRecommendationRunCmd.Flags().String(
+		"clickhouse-endpoint",
+		"",
+		"The ClickHouse Service endpoint. (Only works when wait is enabled)",
+	)
+	policyRecommendationRunCmd.Flags().Bool(
+		"use-cluster-ip",
+		false,
+		`Enable this option will use ClusterIP instead of port forwarding when connecting to the ClickHouse Service.
+It can only be used when running in cluster. (Only works when wait is enabled)`,
```

Comment:
What does "(Only works when wait is enabled)" mean?

Should we check "wait" is enabled and say: "It can only be used when running in cluster and when wait is enabled"?

Reply:
Thanks addressed.